plugins/hls-overloaded-record-dot-plugin/test/testdata/NestedParens.hs

```diff
@@ -0,0 +1,16 @@
+data Happy = Happy {name :: String, age :: Int, happy :: Bool}
+
+data Building = Building {address :: String, owner :: Happy}
+
+main :: IO ()
+main = do
+    putStrLn test
+
+man :: Happy
+man = Happy {name = "Happy", age = 1, happy = True}
+
+home :: Building
+home = Building {address = "No. 10 Privet Dr.", owner = man}
+
+test :: String
+test  = name (owner home)
```

Comment:
Given there are two records and the "nested" in the name, was this meant to test two levels of record selection?

Reply:
Was intending to test the name selector, but now since I automatically accept any HsApp expr thought it might not be as important.